### PR TITLE
Tests via docker and in CI

### PR DESCRIPTION
**Pull request by lablans**

### CODE REVIEW

**Review by TKussel — APPROVED** [review]
This PR is a mix of CI/CD related changes and a proxy detection fix. However, to bring this fix into production quickly, I would vote for merging this PR without further history cleanup.

File: proxy/src/serve_tasks.rs [hunk 1]
```diff
@@ -84,7 +84,10 @@ async fn handler_tasks(
 async fn sign_request(req: Request<Body>, config: &config_proxy::Config, target_uri: &Uri, sender: &AppId) -> Result<Request<Body>, (StatusCode, &'static str)> {
     let (mut parts, body) = req.into_parts();
     let body = body::to_bytes(body).await
-        .map_err(|_| ERR_BODY)?;
+        .map_err(|e|{
+            warn!("Unable to read message body: {e}");
+            ERR_BODY
+        })?;
```
Comment (TKussel) [inline proxy/src/serve_tasks.rs:90]: Thank you for the more detailed error feedback. This was a bit of a pain before.

File: README.md [hunk 2]
```diff
@@ -227,6 +227,29 @@ HTTP/1.1 200 OK
...
+./dev/beamdev start
+```
+
+Steps may fail and ask you to install tools. In particular, note that you need a current (>= 7.7.0) curl version.
+
+Alternatively, you can run the services in the background and get the logs as follows:
+
+```shell
+./dev/beamdev start_bg
+docker-compose
```
Comment (torbrenner) [inline README.md:246]: missing "logs" after "docker-compose"?

Comment (lablans) [reply README.md]: Thanks, fixed in b0aad6c

**Pull request merged**